hell/entries/33-make-me-one-input-with-everything.md

```diff
@@ -26,11 +26,10 @@ You might have heard the joke about the Dalai Lama who walks into a pizza shop s
 ## Issues and how to fix them
 
 
-1. The `aria-label`, `placeholder`, and `title` attributes all provide similar or identical information ("First name"), leading to redundancy and potential confusion.
-1. The `aria-label` is unnecessary in this context since the input is already correctly labeled by the `<label>` element. This can lead to conflicts or confusion for assistive technology users.
+1. The `aria-label`, `placeholder`, and `title` attributes all provide the same informaton ("First name"), leading to the screenreader reading the same text multiple times.
```

Comment:
For real? I thought that the accessible name provided by `aria-label` overrides the accessible name provided by `placeholder` which overrides the accessible name provided by `title`!?

Reply:
On VoiceOver on macOS Sonoma it reads the aria-label followed by the placeholder. NVDA did the same. I modified the example in order to have a better test: https://codepen.io/MoritzGiessmann/pen/KKjevNO

Reply:
My bad, you're right!